src/java/org/apache/cassandra/io/sstable/indexsummary/IndexSummaryManager.java

```diff
@@ -114,10 +122,24 @@ public int getResizeIntervalInMinutes()
         return DatabaseDescriptor.getIndexSummaryResizeIntervalInMinutes();
     }
 
+    private static int getDurationInMinutes(DurationSpec.IntMinutesBound duration)
+    {
+        return duration == null ? -1 : duration.toMinutes();
+    }
+
     public void setResizeIntervalInMinutes(int resizeIntervalInMinutes)
     {
-        int oldInterval = getResizeIntervalInMinutes();
-        DatabaseDescriptor.setIndexSummaryResizeIntervalInMinutes(resizeIntervalInMinutes);
+        runExceptionally(() -> DatabaseDescriptor.setIndexSummaryResizeIntervalInMinutes(resizeIntervalInMinutes),
+                         e -> {
+                             logger.error("Unable to update index_summary_resize_interval", e);
+                             throw new RuntimeException(e.getMessage());
+                         });
+    }
+
+    private void handleResizeIntervalInMinutes(DurationSpec.IntMinutesBound oldValue, DurationSpec.IntMinutesBound newValue)
+    {
+        int oldInterval = getDurationInMinutes(oldValue);
+        int resizeIntervalInMinutes = getDurationInMinutes(newValue);
```

Comment:
@Mmuzaf where was this property taken from before? I see it is already used in this method further down bellow ...